hack/bundle/render_templates.sh

```diff
@@ -28,9 +28,6 @@ update_csv_manifest()
 	sed -i "s#quay.io/jetstack/cert-manager-cainjector.*#${CERT_MANAGER_CA_INJECTOR_IMAGE}#g" "${CSV_FILE}"
 	sed -i "s#quay.io/jetstack/cert-manager-acmesolver.*#${CERT_MANAGER_ACMESOLVER_IMAGE}#g" "${CSV_FILE}"
 
-	## replace kube-rbac-proxy image
-	sed -i "s#gcr.io/kubebuilder/kube-rbac-proxy.*#${KUBE_RBAC_PROXY_IMAGE}#g" "${CSV_FILE}"
-
```

Comment:
https://github.com/openshift/cert-manager-operator-release/pull/191/files#diff-d6e9bb93c51dd8fa0dfcfa1969ce22b27a6c6ce4755c9fbf65f28e06754d98f3R12
https://github.com/openshift/cert-manager-operator-release/pull/191/files#diff-d6e9bb93c51dd8fa0dfcfa1969ce22b27a6c6ce4755c9fbf65f28e06754d98f3R59
needs to be removed.